### PR TITLE
Set-AzAlertProcessingRule support 24-hour clock

### PR DESCRIPTION
**Pull request by lijinpei2008**
## Description
Checking ScheduleEndDateTime and Rule]support 24-hour clock

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/AlertsManagement/AlertsManagement/ActionRuleCommands/SetAzureAlertProcessingRule.cs [hunk 1]
```diff
@@ -677,7 +677,7 @@ private void ValidateSchedule()
             {
                 if (System.DateTime.TryParseExact(ScheduleStartDateTime, format, CultureInfo.InvariantCulture, DateTimeStyles.None, out outDateTime) == false)
                 {
-                    throw new PSInvalidOperationException("Invalid ScheduleStartDateTime Format");
+                    throw new PSInvalidOperationException("Invalid ScheduleEndDateTime Format");
```
Comment (isra-fel) [inline src/AlertsManagement/AlertsManagement/ActionRuleCommands/SetAzureAlertProcessingRule.cs:680]: Line 678 is checking ScheduleStartDateTime, why was this 680 throwing error about ScheduleEndDateTime?

Comment (lijinpei2008) [reply src/AlertsManagement/AlertsManagement/ActionRuleCommands/SetAzureAlertProcessingRule.cs:680]: The if statement at line 676 validates the ScheduleEndDateTime parameter

Comment (lijinpei2008) [reply src/AlertsManagement/AlertsManagement/ActionRuleCommands/SetAzureAlertProcessingRule.cs:680]: ScheduleStartDateTime on line 678 has been updated to ScheduleEndDateTime

**Review by isra-fel — APPROVED** [review]

**Pull request merged**